WebHostLib/templates/header/baseHeader.html

```diff
@@ -34,12 +32,12 @@
         </div>
     </header>
     <div id="base-header-mobile-menu">
-        <a href="/games">supported games</a>
+        <a class="first-link" href="/games">supported games</a>
```

Comment:
as above

Reply:
That's a better idea, in case they get changed later.